app/views/recipes/_table.html.erb

```diff
@@ -0,0 +1,29 @@
+<table class="table table-striped my-3">
+  <thead>
+    <tr>
+      <th scope="col">#</th>
+      <th scope="col">Food</th>
+      <th scope="col">Quantity</th>
+      <th scope="col">Value</th>
+      <th scope="col">Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @food_recipes.each_with_index do |food_recipe, index| %>
+      <tr>
+        <th scope="row"><%= index %></th>
+        <td><%= food_recipe.food.name %></td>
+        <td><%= food_recipe.food.name_with_measurement_unit %></td>
```

Comment:
- [ ] @Team Great work on displaying the ingredients, however, this table data should display the actual quantity of food items eg 2 grams 2 kg, like in the wireframe design 
![image](https://github.com/Estete9/recipes-app/assets/61408860/33efd552-2c90-47c4-9c81-27128f18ab4f)